src/cloud/cdn/cloud-vcl-custom-snippets.md

```diff
@@ -43,7 +43,7 @@ The examples and instructions in the {{ site.data.var.ece }} documentation expla
 
 You can create and manage custom VCL snippets from the Magento Admin UI or by using the Fastly API:
 
--  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI.
+-  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting snippets from the Magento UI only removes it from the Magento UI. If you really need a snippet gone you will need to delete it in the Fastly UI. Alternative to deleting a snippet is to set the VCL contents to empty content and upload VCL.
```

Comment:
Cloud projects do not get access to Fastly UI. Everything has to be done via the Magento admin. This note needs to be updated to reflect this.